public/app/app.jsx.BACKUP.32588.jsx

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+var React = require('react');
+var Reflux = require("reflux");
+
+<<<<<<< HEAD
```

Comment:
Artem, I want you to review your own pull request first before presenting it to others.